### PR TITLE
feat(opl3lpt): omit adlib detection and direct call to OPL3LPT

### PR DESCRIPTION
**Pull request by jsmolina**
Now it's just rewiring any Adlib call to OPL3LPT. Proper implementation implies modifying FDSETUP code.

### CODE REVIEW

**Comment by viti95** [comment]
Does this work on a real OPL3LPT?

**Comment by jsmolina** [comment]
> Does this work on a real OPL3LPT?
@viti95 
yes!!! I just tried with a real one😄 
[FDOOM13H.zip](https://github.com/jsmolina/FastDoom/files/9734761/FDOOM13H.zip)

File: FASTDOOM/p_sight.c [hunk 1]
```diff
@@ -262,9 +262,9 @@ byte P_CheckSight(mobj_t *t1, mobj_t *t2)
     // First check for trivial rejection.
 
     // Determine subsector entries in REJECT table.
-    s1 = (t1->subsector->sector - sectors);
-    s2 = (t2->subsector->sector - sectors);
-    pnum = s1 * numsectors + s2;
+    pnum = Div96((int)t1->subsector->sector - (int)sectors);
```
Comment (jsmolina) [inline FASTDOOM/p_sight.c]: this was accidental change ported from old version, I will remove it later.

**Comment by jsmolina** [comment]
> Does this work on a real OPL3LPT?

This is the demonstration of it working. It's nice as it sound in the real stereo OPL3. SFX is coming from my Solo1 emulating a Sound blaster, but OPL3 is coming from the LPT port.
https://www.youtube.com/watch?v=x__xsLXnV3c

So to make it work, it's just a matter of changing the ports to LPT1, on Left, Right and Adlib. Then just doing a call to your prepared code to Output for OPL3LPT and it works like a charm.
Adlib detection of course will fail, so it needs to be skipped.

**Comment by viti95** [comment]
Great! I didn't spend much time adding support to the OPL2LPT and OPL3LPT devices since I don't own them, it's great to hear the base code was basically fine and it only required to be connected.

**Pull request merged**